code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -658,11 +658,11 @@
 	construction_time = 120
 	category = list("Cyborg Upgrade Modules")
 
-/datum/design/borg_upgrade_ashplating
-	name = "Cyborg Upgrade (Ash Storm Plating)"
-	id = "borg_upgrade_ashplating"
+/datum/design/borg_upgrade_lavaproof
+	name = "Cyborg Upgrade (Lavaproof Tracks)"
+	id = "borg_upgrade_lavaproof"
 	build_type = MECHFAB
-	build_path = /obj/item/borg/upgrade/ashplating
+	build_path = /obj/item/borg/upgrade/lavaproof
 	req_tech = list("plasmatech" = 4, "materials" = 4, "engineering" = 4)
 	materials = list(MAT_METAL = 8000, MAT_PLASMA = 10000)
```

Comment:
The reason your pr is taking so long to merge is all the crap you have in it.

I would have happily merged a pr containing the fixes,  a pr containing QOL changes and a pr containing the borg stuff ages ago, as it is, I'm unwilling to touch it because of the explosion changes because I know at least some people will get mad about it.